setup.py

```diff
@@ -143,9 +143,11 @@ def run(self):
         package_dir={'pybitmessage': 'src'},
         packages=packages,
         package_data={'': [
-            'bitmessageqt/*.ui', 'bitmsghash/*.cl', 'sslkeys/*.pem', 'bitmessagekivy/main.kv',
-            'bitmessagekivy/kv/*.kv', 'translations/*.ts', 'translations/*.qm', 'default.ini', 'sql/*.sql',
-            'images/*.png', 'images/*.ico', 'images/*.icns', 'bitmessagekivy/screens_data.json'
+            'bitmessageqt/*.ui', 'bitmsghash/*.cl', 'sslkeys/*.pem',
+            'translations/*.ts', 'translations/*.qm', 'default.ini', 'sql/*.sql',
+            'images/*.png', 'images/*.ico', 'images/*.icns',
+            'bitmessagekivy/main.kv', 'bitmessagekivy/screens_data.json',
+            'bitmessagekivy/kv/*.kv', 'bitmessagekivy/tests/sampleData/*.dat'
```

Comment:
you can do the `extend` inside the `if` condition up where you test for `INSTALL_TESTS`